README.md

```diff
@@ -163,8 +163,30 @@ python setup.py nosetests
 This library requires at least Python 2.6, but otherwise has no
 external dependencies.
 
+
 The library does assume that genome sequence is available through a `pygr`
 compatible `SequenceFileDB` object. For an example of writing a wrapper for
 a different genome sequence back-end, see
 [hgvs.tests.genome.MockGenome](hgvs/tests/genome.py).
 
+## Flask Web Service
```

Comment:
Thanks for this contribution! 

Do you think this could be done as a separate package that imports the hgvs library? Majority of use cases will require just the library, not the HTTP API.

Reply:
Ho @pkaleta totally agree, I can put this as a separate one.  When I originally wrote this it was because our pipeline was primarily java and wanted a way to interact with this.  Alternatively this can be a multi-package project.  Up to you.  

Let me know.